package.json

```diff
@@ -277,8 +277,8 @@
     "@types/react-redux": "6.0.4",
     "@types/redux-logger": "^3.0.7",
     "awesome-typescript-loader": "^5.2.1",
-    "brave-ui": "github:brave/brave-ui#44513d1b6c27f4bcdf2564baef092af9fabf5665",
-    "css-loader": "^2.1.1",
+    "brave-ui": "github:brave/brave-ui#974bb2d36ac32afd90b65710414f3e9f74d18d27",
+    "css-loader": "^0.28.9",
```

Comment:
why do we need to update this dep in this PR?

Reply:
We do not, this was a mistake, undone